clients/go/pkg/zbc/oauthCredentialsCache_test.go

```diff
@@ -86,7 +86,9 @@ func (s *oauthCredsCacheTestSuite) TestWriteCacheGoldenFile() {
 	err = cache.Update(capybaraAudience, capybara)
 	s.NoError(err)
 
-	cacheCopy, _ := NewOAuthYamlCredentialsCache(cachePath)
+	cacheCopyI, _ := NewOAuthYamlCredentialsCache(cachePath)
```

Comment:
We could also have `NewOAuthYamlCredentialsCache` wrap a `newOAuthYamlCredentialsCache` which returns the unexpected struct, and us that here. IDK, I don't care that much, this is also fine for me